learners/BF_measles.Rmd

```diff
@@ -0,0 +1,884 @@
+---
+
+# Preface
+
+This document is prepared for the participants of the Outbreak Analytics in R course, 
+organized by the Epiverse-TRACE team. Its purpose is to provide a concise 
+introduction to the mathematical theory behind modeling infectious epidemics. 
+We will use the Susceptible, Exposed, Infected, and Recovered (**SEIR**) model, 
+applicable to diseases like measles, mumps, rubella, as an example.
+
+# Scenario
+
+Suppose a new infection has been detected in a closed community of
+$\mathbf{N}$ individuals, with the infection diagnosed in a single
+individual. The remaining $\mathbf{N-1}$ individuals are all at risk of
+getting the infection and are considered susceptible ($\mathbf{S}$). An
+infected person interacts, on average per time unit (e.g., a day, week,
+or month), with some $\mathbf{k}$ individuals within the community. In
+particular, the infected person interacts with
+$\frac{\mathbf{K\cdot S}}{\mathbf{N}}$ susceptible individuals, among
+whom $\mathbf{k}' \leq \mathbf{k}$ may become infected. If the infection
+is highly contagious, $\mathbf{k}'$ will be closer to $\mathbf{k}$, and
+vice versa. The probability of transmission is calculated as
+$\mathcal{P}= \frac{\mathbf{k}'}{\mathbf{k}}.$ Consequently, on
+average, the infected individual will infect
+$\beta\cdot \frac{\mathbf{S}}{\mathbf{N}}$ susceptible individuals per
+time unit, where $\beta = \mathcal{P} \cdot \mathbf{k}$ is called the
+effective transmission rate. This represents the average rate at which
+an infected individual can transmit the infection to a susceptible
+individual.
+
+Depending on the nature of the disease, a newly infected individual may
+take time to show symptoms. This time period is known as the latent
+period. Susceptible individuals who come into contact with an infected
+person and acquire the infection first move to an intermediate state
+called exposed ($\mathbf{E}$). T here they (he/she) spent, on average,
+$\frac{1}{\omega}$ units time (the latent period) and then move to
+the infected class ($\mathbf{I}$) at a constant rate $\omega$. Infected
+individuals are removed (either by recovery or death) from $I$ at a
+constant rate $\gamma$, with $\frac{1}{\gamma}$ representing the average
+recovery duration. Upon recovery, they gain lifelong immunity against
+the current infection, meaning they do not become susceptible again, and
+they move to the recovered class ($\mathbf{R}$) (see
+[Figure 1](#fig-seir)).
+
+![](fig/t0.drawio.png){ref-parent="fig-seir"}
+![](fig/t1.drawio.png){ref-parent="fig-seir"}
+![](fig/t2.drawio.png){ref-parent="fig-seir"}
+![](fig/t3.drawio.png){ref-parent="fig-seir"}
+
+The four classes considered define the $\mathbf{SEIR}$ model. The sizes
+of these classes change overtime and thus are functions of time $t$, and
+the total population at any time $t$ is
+$N(t)=  S(t) + E(t) + I(t)+ R(t)$. The rates of change between these
+classes are captured mathematically through four differential equations
```

Comment:
No or with little knowledge

Reply:
So ODE equations might be quite a barrier to this audience? Wonder if we should include an optional primer box (e.g. to break down componenets, and make it clear that dS/dt is the rate at which number of susceptible individuals changes over time, and can write this in terms of people who enter/leave the compartment)?

From experience, applied learners can find the idea of fractions of individuals in compartments and rates a bit non-intuitive, so worth a bit of extra content here I think to avoid losing people.

Reply:
Good idea. This will be addressed in issue #60